python/requirements.txt

```diff
@@ -1,6 +1,6 @@
-msgpack==1.0.0
-loguru==0.5.3
-requests==2.24.0
-requests-http-signature==0.2.0
-websocket-client==0.57.0
-cryptography==3.2
+msgpack==1.*
+loguru
+requests==2.*
```

Comment:
I think this is a little too relaxed, we haven't tested with old requests versions, and it's not impossible that they wouldn't work, particularly with signatures.